lib/KalturaHelpers.php

```diff
@@ -217,6 +217,14 @@ public static function getOption( $name, $default = null ) {
 		}
 	}
 
+	private static function isPluginNetworkActivated() {
+		if ( ! function_exists( 'is_plugin_active_for_network' ) ) {
+			require_once( ABSPATH . '/wp-admin/includes/plugin.php' );
+		}
+
+		return is_plugin_active_for_network( 'all-in-one-video-pack.wordpress/all-in-one-video-pack.php' );
```

Comment:
This is problematic, you are relying on `all-in-one-video-pack.wordpress` as the plugins' folder.